CueSheeter/jsx/Premiere.jsx

```diff
@@ -88,7 +89,7 @@ $._AA_={
                          currentChild = currentItem.children[j+1];
                          if (currentChild){
                              return $._AA_.searchBinForProjItemByName(0, currentChild, nameToFind);
-                         } else 
```

Comment:
@bbb999 wanted to double check this change is fine. My editor was throwing an error since this is an extra `else`